Sources/Persister/Utilities/DiskManager.swift

```diff
@@ -122,21 +122,25 @@ extension FileAttributeKey {
 private extension FileManager {
     
     func extendedAttribute(_ name: String, on path: String) throws -> Data {
-        // Get size of attribute data
-        var size = getxattr(path, name, nil, 0, 0, 0)
-        if size == -1 {
+        // Get size of attribute data. https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/getxattr.2.html
+        let size = getxattr(path, name, nil, 0, 0, 0)
+        
+        guard size > 0 else {
```

Comment:
This change kind of surprises me, specifically because the docs say it returns -1 on failure, which is what we were checking for before